frontend/packages/client/src/components/Community/CommunityPropsAndVoting.js

```diff
@@ -0,0 +1,177 @@
+import React, { useState } from "react";
+import { useVotingStrategies } from "hooks";
+import { AddButton, ActionButton } from "components";
+import { Bin } from "components/Svg";
+import { useModalContext } from "contexts/NotificationModal";
+import StrategyEditorModal from "./StrategyEditorModal";
+
+const StrategyInput = ({ index, commuVotStra, onDeleteStrategy } = {}) => {
```

Comment:
Could be moved as a separated component. We have a lot of inputs like this one, we can plan to have a separate Pr to try to reuse a common component